autogalaxy/aggregator/agg_util.py

```diff
@@ -1,25 +1,66 @@
 from __future__ import annotations
 from typing import List, Optional
 
+from autoconf.fitsable import ndarray_via_hdu_from
+
 import autofit as af
 import autoarray as aa
 
 from autogalaxy.analysis.adapt_images.adapt_images import AdaptImages
 
 
+def mask_header_from(fit):
+    """
+    Returns the mask, header and pixel scales of the `PyAutoFit` `Fit` object.
+
+    These quantities are commonly loaded during the aggregator interface therefore this method is used to
+    avoid code duplication.
+
+    Parameters
+    ----------
+    fit
+        A `PyAutoFit` `Fit` object which contains the results of a model-fit as an entry which has been loaded from
+        an output directory or from an sqlite database.
+
+
+    Returns
+    -------
+    The mask, header and pixel scales of the `PyAutoFit` `Fit` object.
+    """
+
+    header = aa.Header(header_sci_obj=fit.value(name="dataset")[0].header)
+    pixel_scales = (
+        header.header_sci_obj["PIXSCAY"],
+        header.header_sci_obj["PIXSCAX"],

```

Comment:
If you use these keys in more than one place consider creating an enum or similar.

class FITSHeaders(Enum):
    PIXSCAY = "PIXSCAY"

header.header_sci_obj[FITSHeader.PIXSCAY.value]
```